Classes/Command/DatabaseStorageCommandController.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+namespace Wegmeister\DatabaseStorage\Command;
+
+use DateInterval;
+use DateTime;
+use Wegmeister\DatabaseStorage\Service\DatabaseStorageService;
+use Neos\Flow\Annotations as Flow;
+use Neos\Flow\Cli\CommandController;
+
+/**
+ * @Flow\Scope("singleton")
+ */
+class DatabaseStorageCommandController extends CommandController
+{
+    /**
+     * @var DatabaseStorageService
+     * @Flow\Inject
+     */
+    protected DatabaseStorageService $databaseStorageService;
+
+    /**
+     * @Flow\InjectConfiguration(package="Wegmeister.DatabaseStorage", path="cleanup")
+     * @var array|null
+     */
+    protected ?array $storageCleanupConfiguration;
+
+    /**
+     * Deletes entries of configured storages older than configured date interval
+     */
+    public function cleanUpConfiguredStoragesCommand(): void
+    {
+        $this->outputFormatted('<b>Cleanup of configured storages</b>');
+        $this->outputLine('');
+
+        if (empty($this->storageCleanupConfiguration)) {
+            $this->outputFormatted('No cleanup configuration found.', [], 4);
+            $this->outputFormatted('Please configure the cleanup for Wegmeister.DatabaseStorage in Settings.yaml.', [], 4);
+            return;
+        }
+
+        $results = [];
+        foreach ($this->storageCleanupConfiguration as $storageIdentifier => $storageCleanupConfiguration) {
+            $results[$storageIdentifier] = ['storageIdentifier' => $storageIdentifier, 'messages' => ''];
```

Comment:
`foreach` could theoretically be merged with the `foreach` in `cleanupAllStoragesCommand()`.